02/app.js

```diff
@@ -0,0 +1,59 @@
+//Style
+
+const functionStyle = {
+    border: '3px solid black',
+    backgroundColor: 'yellow',
+    color:'red',
+    borderradius: '5px',
+};
+
+
+
+//Rozwiązanie przy wykorzystaniu funkcji
+
+import React from 'react';
+import {createRoot} from 'react-dom/client';
+
+const root = createRoot(document.querySelector('#root'));
+
+const getText = () => 'Moja pierwsza strona w React !';
+
+root.render(
+    <headerFn
+        style ={functionStyle}
+    >
+        {getText()}
+    </headerFn>
```

Comment:
Jeśli wprowadzisz zmiany, o których piszę wyżej to tutaj uruchomienie powinno wyglądać tak:
````
root.render(<Text/>);
```